cmd/rancher_release/list_images_rc.go

```diff
@@ -0,0 +1,35 @@
+package main
```

Comment:
This file needs to be formatted. Might need to check your IDE settings to see why this isn't happening automatically.

Reply:
Actually, it appears as though all files in this PR need to be formatted.

Reply:
Damn neovim

Reply:
It appears as though the file still needs to be formatted.

Reply:
I used the auto formatter, so I think it is already formatted.

Reply:
There's indentation of 2 space and 4 spaces in different locations. Something is wrong.